### DISCUSSION TITLE
DNS Flood Requests to blacklisted site [Edit: Not related to Tasmota]

### DISCUSSION THREAD

**Discussion by stephennutt** [arendst/Tasmota#13219] [Q&A] [answered]
About 48 hours ago, my Sonoff Bridge started making thousands of DNS Queries to neptune.mobileposse.com; over 40,000 in the last 24 hours.  My PiHole is blocking them all.  Has anyone else had something like this maliciously installed on a Tasmotized device?  It started about the same time as I was installing Nagios on a Raspberry Pi although it could be a complete coincidence. I have since removed Nagios but nothing changed with the DNS queries.

I tried using the Reset 2 command to reset device hoping that would make it stop with no luck.  Next step is to re-flash with Tasmota software but I was hoping to avoid breaking out the equipment to do so and going back to Ground Zero.  

Has anyone else experienced something like this?

**Comment by barbudor** [comment]
Very strange
When you say DNS queries, do you mean Tasmota inquires your DNS about `neptune.mobileposse.com` ? 
Where does your Tasmota binary comes from ? Official ? custom built ? Somewhere else ?

**Reply by barbudor** [reply]
I would very doubt that Tasmota get infected. I would be the 1st time that we see a trojan targetting Tasmota.

There are some elements on the Web about neptune.mobileposse.com which seems to state it is a malicious site
But most of them refers to infected mobile phone
Are you sure the request comes from a tasmota device ?
Do they stop when you turn off the device ?
Could they come from an infected mobile/PC that has a Tasmota web page open (even including TasmoAdmin) ?

**Comment by stephennutt** [comment]
I might be using the wrong terminology.  This is what I am seeing in PiHole:

![image](https://user-images.githubusercontent.com/25011547/134774707-c300b5a5-be49-42c5-a5da-b136f56819f9.png)


![image](https://user-images.githubusercontent.com/25011547/134774735-7ded7fd5-e8b2-4931-98f1-50b9df056c54.png)


It may be some other device with the issue but PiHole is sure making it look like it is coming from the Bridge.

**Reply by stephennutt** [reply]
Ok, turned off the devce and deleted sonoff.bridge from the list of local DNS domains to see if the queries would stop or change to the Bridge's IP address.  PiHole is still showing queries coming from "sonoff.bridge" even after making those changes so this must be spoofing the bridge somehow.  VERY STRANGE!!!

**Reply by barbudor** [reply]
You have another device with the same IP as your Sonoff.bridge
Conflict of DHCP vs fixed IP ?

By the way, if you plan to use Nagios in relation with Tasmota and home automation, we would be happy to get your impression, and feedback, may be as a HOW-TO in this Discussion forum.

**Comment by ascillato** [comment]
Hi, I can't reproduce your issue. Are you 100% sure that you are using official Tasmota firmware from http://ota.tasmota.com/tasmota/tasmota.bin.gz ?

**Reply by Jason2866** [reply]
It is not Tasmota. A device which is powered off can not make IP traffic. You have a other device in your network which is going "mad".

**Reply by stephennutt** [reply]
Yes, definitely using official firmware.  Based on my additional research triggered by @barbudor 's comments, I think this isn't a Tasmota issue and is probably something else.  I am going to try some other communities but, if I get a resolution, I will post back here.  Thanks for everyone's quick response to help.

**Reply by ascillato2** [reply]
Yes, please, as soon as you have any hints, tell us. It is interesting to know where your DNS flooding is coming from.

I bet a laptop or a phone.

**Comment by ascillato** [comment]
Ok, so as this security issue (flooding DNS) is not related to Tasmota, let's close it. Thanks.

**Comment by stephennutt** [comment, marked as answer]
**IT WAS A PHONE!!!!!** that I use simply as a dashboard for my home automation system.  I started turning devices off and the flooding stopped when I turned off this phone.  Then, I found this listing that contained "neptune.mobileposse.com":  
https://discourse.pi-hole.net/t/80-blocked-neptune-mobileposse-com/49763

The culprit was a Discover Bar app installed on my Motorola phone also.  I don't remember ever installing it as I really only have a few apps because I don't use it as a phone as previously mentioned.  Still weird that it was somehow spoofing all of these queries and making PiHole think that it was my Tasmotized Sonoff Bridge sending all of them.  Thanks again for everyone who was quick to jump in with help.

**Reply by barbudor** [reply]
As I wrote above, it may be an IP address conflict between DHCP and fixed IP

**Reply by stephennutt** [reply]
The Sonoff Bridge is on my IoT VLAN and the phone in question is on my Main network.
